src/App.jsx

```diff
@@ -14,45 +15,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [status, setStatus] = useState({
+    alphabetical: false,
+    bylength: false,
+    reversed: false,
+  });
```

Comment:
```suggestion
  const [status, setStatus] = useState({
    sort: '',
    isReversed: false,
  });
```
suggest to simplify logic. no need to manage both sort fields in state (let's imagine that we have 10 sort fields). it can be value with string type ('alphabetical' or 'bylength' and '' as default). 
also instead one status state can be two separate state for sort and isReversed (but can be in one state)